src/bokeh/layouts.py

```diff
@@ -314,7 +314,7 @@ def merge(cls: type[Tool], group: list[Tool]):
     active_taps = [ toolbar.active_tap for toolbar in toolbars ]
     active_multis = [ toolbar.active_multi for toolbar in toolbars ]
 
-    T = TypeVar("T")
+    TypeVar("T")
```

Comment:
I suppose this could go behind `TYPE_CHECKING`

Reply:
This was actually incorrect, picking up the global type var defined later in the file.